### PR TITLE
feat(intake): add governed incoming intake routing

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: .octon/framework/execution-roles/practices/pull-request-standards.md
Reminder: Run @codex review.

## What

Adds Mature Governed Incoming Intake Routing as the additive .incoming admission layer inside process-incoming-intake, with four route outcomes, versioned decision/handoff/admission contracts, proposal-owned intake admission contracts, and fixture-backed validators.

## Why

No-Issue: operator-requested governance implementation.

This reduces additive intake ambiguity while preserving authority boundaries: raw intake remains non-authoritative, proposal/program lifecycles own admission and receipts, and closeout-worktree/repo-hygiene cleanup stay target-owned follow-on routes.

## How

The intake workflow now validates the raw envelope first, then classifies into single-work-unit-handoff, coordinated-program-handoff, disabled target-owned-direct-handoff, or blocked-rejected-deferred. The implementation adds JSON schemas, proposal packet/program admission contracts, lifecycle forbidden-transfer coverage for worktree cleanup and repo hygiene deletion, product feature documentation, fixture matrix expansion, and static validator coverage. Real target dispatch remains disabled until later contract-backed rollout phases.

## Profile Selection Receipt

- Semantic version source(s): repository pre-1.0 governance posture
- Release state (pre-1.0 or stable): pre-1.0
- change_profile (atomic or transitional): atomic
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): no runtime dispatch enabled; no migration/backfill; governance/docs/contracts/validators only; blast radius limited to intake workflow documentation and static validation surfaces
- Tie-break status: n/a
- transitional_exception_note (required when pre-1.0 + transitional): n/a

## Implementation Plan

- Workstreams and scope: contracts, intake governance docs, process-incoming-intake workflow docs, proposal lifecycle admission contracts, product feature catalog, static validation fixtures/tests
- Public interfaces/types/contracts affected: governed-incoming-intake-route-decision-v1, governed-incoming-intake-handoff-v1, target-owned-intake-admission-contract-v1, lifecycle-interaction-request-v1 forbidden transfers
- Test scenarios: positive packet/program handoffs plus invalid envelope, ambiguity, unsafe provenance, secrets/private data, authority confusion, direct target without contract, stale returns, requested-route mismatch, scope drift, parent/child receipt isolation, lifecycle receipt authorization misuse
- Assumptions/defaults: proposal packet/program handoff is advisory and target-owned; direct target handoff remains disabled; no real proposal creation, GLO dispatch, install, archive, closeout, worktree cleanup, or repo hygiene deletion is enabled

## Impact Map (code, tests, docs, contracts)

- Contracts: .octon/framework/product/contracts/governed-incoming-intake-*.json, target-owned-intake-admission-contract-v1, lifecycle-interaction-request-v1
- Workflow/docs: additive incoming intake governance, process-incoming-intake command/workflow/stages, governed incoming intake routing feature doc and catalog entry
- Proposal lifecycle: proposal packet/program intake admission contracts and lifecycle forbidden-transfer declarations
- Assurance: governed incoming intake routing fixture matrix, validator, shell test, lifecycle validators

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (atomic) unless hard gates required transitional.
- [x] Included transitional_exception_note when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

This is intentionally a mature contract and fixture foundation, not behavior-changing dispatch. Direct raw installation remains current behavior until a later migration PR changes workflow semantics. Autonomous handoff remains out of scope until target-owned contracts, receipts, validators, rollback posture, and negative fixtures prove it safe.

## Testing

- bash .octon/framework/assurance/runtime/_ops/scripts/validate-governed-incoming-intake-routing.sh
- bash .octon/framework/assurance/runtime/_ops/tests/test-validate-governed-incoming-intake-routing.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-product-feature-catalog.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-lifecycle-contracts.sh
- bash .octon/framework/assurance/runtime/_ops/tests/test-lifecycle-interaction-receipts.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-lifecycle-interaction-receipts.sh --self-test
- bash .octon/framework/assurance/runtime/_ops/tests/test-validate-incoming-intake-unit.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-input-non-authority.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-raw-input-dependency-ban.sh
- bash .octon/framework/assurance/runtime/_ops/scripts/validate-workflow-authority-derivation.sh
- bash -n relevant shell scripts
- git diff --check HEAD~1..HEAD

## Rollout

Draft PR first. Merge only after review and required checks. Later PRs should add workflow migration, proposal admission mock execution, supervised handoff, GLO integration, and any direct target route only after target-owned contracts exist.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert commit f3c6a0aad or revert this PR before enabling behavior-changing dispatch
- Rollback handle: f3c6a0aad
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes: adds governed incoming intake route decision, handoff, and target-owned admission schemas; extends lifecycle interaction forbidden transfers
- Threat model update/link: authority-confusion, secrets/private data, unsafe provenance/licensing, parent/child receipt isolation, lifecycle receipt misuse, stale target return, requested-route mismatch, and scope-drift fixtures added

## Observability and Performance

- Traces/logs/metrics for changed flows: retained route decision, denial, handoff, and target return evidence requirements documented
- Representative traces for high risk changes: fixture matrix covers route and denial evidence
- Performance or bundle impact: n/a

## License and Provenance

- New dependencies and licenses: none
- Generated code/templates provenance notes: none

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] Reviewer feedback addressed with fix, commit, push, reply; reviewer-owned threads confirmed by reviewer/maintainer or resolved under the documented solo-maintainer exception

## Screenshots / Notes

No visual changes.

### CODE REVIEW

File: .octon/framework/orchestration/runtime/workflows/meta/process-incoming-intake/workflow.yml [hunk 1]
```diff
@@ -104,7 +108,7 @@ stages:
     consumes:
       - stage:classify-route
     produces:
-      - stage:execute-disposition
+      - stage:execute-handoff-or-denial
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/orchestration/runtime/workflows/meta/process-incoming-intake/workflow.yml]: Resolved by subsequent commits. The workflow now consistently produces and consumes `stage:execute-disposition` in `.octon/framework/orchestration/runtime/workflows/meta/process-incoming-intake/workflow.yml`, and this thread is marked outdated. Validation evidence: workflow validation, runtime-effective-state, architecture, and the full PR check set are green at head `2b04ad2309464209631298108ce9e8f23e62bb80`.
